src/deepsparse/transformers/server.py

```diff
@@ -20,9 +20,8 @@
 
 from pydantic import BaseModel, Field
```

Comment:
can be obscure pydantic or does this need to be part of the autoinstall?

Reply:
oh yep, good catch. That makes this a bit harder, but we can get something optional working here